romanesco/format/netcdf/binary_to_dataset.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Binary to Dataset",
+    "inputs": [{"name": "input", "type": "netcdf", "format": "binary"}],
+    "outputs": [{"name": "output", "type": "netcdf", "format": "dataset"}],
```

Comment:
should we call it netcdf-dataset? dataset may be too generic but I am fine with calling it just dataset.